spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'User index page', type: :feature do
+  before :each do
+    @user1 = User.create(name: 'Tomas', photo: 'https://usp.com/pto.png', bio: 'Web developer')
+    visit root_path
+  end
+
+  it 'displays username of all other users' do
+    expect(page).to have_content(@user1.name)
+  end
+
+  it 'displays profile picture for each user' do
+    expect(page).to have_css('img')
+  end
+
+  it 'the number of posts each user has written' do
+    expect(page).to have_content('Number of posts')
+  end
```

Comment:
- [ ] Since this is an index page we are expected to display all the users in this page. We are expected to display the number of post each user has to its name. Kindly associate the user with the post. For example `all_user.post.count`. Something like that can help

Reply:
@addod19, Do you mean I should sum all post on the page and return the count for one user? 🤔